archetype/package.json

```diff
@@ -7,22 +7,21 @@
     "trailjs"
   ],
   "dependencies": {
-    "trailpack": "latest"
+    "trailpack": "next"
   },
   "devDependencies": {
     "lodash": "^4.11.1",
-    "trails": "latest",
-    "eslint": "^2.4",
-    "eslint-config-trails": "^1.0.4",
-    "mocha": "^2.4.5",
-    "smokesignals": "^1.0.9",
-    "trailpack-core": "^1.0.0-beta-6"
+    "trails": "next",
```

Comment:
This will not work with the current generator, or force every trailpack to have a next tag right ? Really hate this lol ^^ too much complain on the community to this kind of error

Reply:
My bad I didn't notice it was the trailpack repo ^^ ignore this please :)